tools/cpanfile

```diff
@@ -53,6 +49,8 @@ requires 'Minion::Backend::Redis', 0.002;
 # Background Worker (Shinobu)
 requires 'Proc::Simple',       1.32;
 requires 'Parallel::Loops',    0.10;
```

Comment:
The `Parallel::Loops` require can probably be taken out now, right?

Reply:
I left it in case 3rd party plugins use it, same with `split_workload_by_cpu`.

Reply:
Right, `split_workload` is in Utils so it's technically fair game for plugins.. I'll have to remember not to remove it in an eventual cleanup 😬